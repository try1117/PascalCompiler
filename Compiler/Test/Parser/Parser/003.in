program abc;
var
 a,b,c , d:integer;
 c3 : Char;
 e : Double;
 c1, c2 : Char; 
begin
end.