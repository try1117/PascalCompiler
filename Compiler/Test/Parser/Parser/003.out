Syntax exception in position (1,8) - Required VARIABLE but found ;
